docs/date.md

```diff
@@ -496,3 +496,17 @@ date = Temporal.Date.from('2006-08-24');
 Object.assign({}, date).day  // => undefined
 Object.assign({}, date.getFields()).day  // => 24
 ```
+
+### date.**getISOCalendarFields()**(): { year: number, month: number, day: number }
+
+**Returns:** a plain object with properties expressing `date` in the ISO 8601 calendar.
+
+This method is mainly useful if you are implementing a custom calendar.
+Most code will not need to use it.
+Use `date.getFields()` instead.
+
+Usage example:
+```javascript
+date = Temporal.Date.from('2006-08-24');
```

Comment:
Here and below: a more interesting example would be when the calendar is *not* ISO-8601.

Reply:
I've also added this example to the calendar branch.